tox.ini

```diff
@@ -2,36 +2,36 @@
 name = tmd
 testdeps =
     mock
-    nose
-    tqdm
+    pytest
 commands = nosetests
 
 [tox]
 envlist =
-    py{36,37}
-    py36-lint
-    py36-coverage
+    py{37, 38, 39, 310}
+    lint
+    coverage
 
 [testenv]
 deps = {[base]testdeps}
 extras = viewer
-commands = nosetests
+commands = pytest tests
 
-[testenv:py36-lint]
-basepython = python3.6
+[testenv:lint]
+basepython = python3.9
```

Comment:
AFAIK, it is safer to run pylint with the oldest supported version of Python because it lints the code according to the Python version with which it is run. So if we use a feature of Python 3.9 in the code, pylint will not complain while it would if it was run with py3.8 for example. Of course, it is very likely that even if pylint does not complain, the tests will fail anyway in this case. But if this feature is used in a not covered part of the code, it might be missed by the CI.

Reply:
Moved lint to python3.7!